helm-chart/kuberay-apiserver/templates/route.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.route.enabled -}}
+{{- $fullName := include "kuberay-apiserver.fullname" . -}}
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "kuberay-apiserver.labels" . | nindent 4 }}
+  {{- with .Values.route.annotations }}
```

Comment:
Why do we use `with` here instead of `if`? In this YAML file, we have used `if` twice and `with` once. Should we strive for consistency?

Reply:
This again is copied from ingress.yaml and does work